.circleci/config.yml

```diff
@@ -604,12 +660,41 @@ jobs:
       - run:
           name: Publish CocoaPods Podspec
           command: pod trunk push
+      - run:
+          name: Inject new Podspec version to CDN cache
+          description: |
+            Force adding new version to the cache to avoid long running clone operation
+            The following command would take the latest CocoaPods version from cache.
+
+            Alternative is to replace CDN with Specs repository and checkout it for the latest changes. It usually takes 3+ minutes on CircleCI
+          command: |
+            source scripts/utilities/cocoapods.sh
+            pod_inject_cdn_version MapboxMaps "$VERSION"
+      - run:
+          name: Test SPM and CocoaPods integrations per version rule
+          command: scripts/validate-integrations/validate-integrations.sh -d -v "$VERSION"
       - slack/status:
           fail_only: false
           include_visit_job_action: true
           failure_message: ':tests-fail-red-cross: <$CIRCLE_BUILD_URL|Release tag for \`v$VERSION\` failed.>'
           success_message: ':green_circle: <$CIRCLE_BUILD_URL|Release tag for \`v$VERSION\` succeeded!> :tada:'
 
+  release-worktrail-validation:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - parse-release-version-if-available
+      - run:
+          name: Configure .netrc
+          command: |
+            echo "machine api.mapbox.com login mapbox password $SDK_REGISTRY_TOKEN" >> ~/.netrc
+            chmod 0600 ~/.netrc
```

Comment:
Should we have a command for this? Looks like we do it in at least 2 places now.

Reply:
Done.